### ISSUE TITLE
[Accessibility] A11y_AzureDataStudio_Operatortab_ScreenReader: Unnecessarily column mark-up read for the table by Voiceover

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5658]
[Check out Keros](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
 GitHubTags: #A11yMAS; #A11yTCS; #MAS1.3.1; #A11y_AzureDataStudio; #MacOs; #DesktopApppp
**Environment Details:**
Application name: Azure Data Studio 
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32

**Additional Details**
ScreenReader Name: Voice Over
OS version: MacOS Mojave 10.14.5
MAS Reference:MAS1.3.1

**Repro Steps**:
1.Launch Visual Studio code desktop application.
2. Connect to the Database -> Go to the SQL Agent ->proxies tab
3.Start Screen reader as Voice Over.
4.Navigate to table and the blank section in the table..
5.Observe that Screen reader is unnecessarily announcing column for the table.

**Actual:**
Screen reader is unnecessarily announcing column for the table.

**Similar issue is observed for the below pages:**
**1.Jobs:** Navigate to the connected server->Select the database->Navigate to the SQL agent tab->
Navigate to jobs page.
**2.Proxiestab:** Navigate to the connected server->Select the database->Navigate to the SQL agent tab->.Navigate to the proxies tab
**3. Alertstab:** Navigate to the connected server->Select the database->Navigate to the SQL agent tab->Navigate to Alerts tab.

**Expected:**
Screen reader should not read that column which does not have any information present for the user.

**User Impact:**
User will not be able to understand why the screen reader reading that column which is unnecessarily present. As there is no information present in the column for the user, it may create confusion for the user.

**Recommendation:**
Please refer the following snippet link for fixes: https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
MAS1.3.1 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[5658_[Accessibility] A11y_Azure Data Studio_ProxiesTab_Screenreader.pptx](https://github.com/microsoft/azuredatastudio/files/3240247/5658_.Accessibility.A11y_Azure.Data.Studio_ProxiesTab_Screenreader.pptx)

Does this issue occur when all extensions are disabled?: Yes
